src/Calypso-SystemTools-QueryBrowser/ClyBrowserMorph.extension.st

```diff
@@ -51,7 +51,14 @@ ClyBrowserMorph >> spawnQueryBrowserOn: aQuery withState: navigationBlock [
 	aQuery isBoundToEnvironment ifFalse: [ 
 		targetQuery := aQuery withScope: self defaultNavigationScope].
 	
-	^self spawnBrowser: ClyQueryBrowser withState: [ :browser | 
+	targetQuery scope = self systemScope ifTrue: [ 
```

Comment:
Extra note for discussion:
In scoped full browser it will always show empty query browser because it is possible that there are some items in global scope.
So the question should we handle it in better way? 
For example In this case we can check global scope too and do not show browser when query is still empty. But it can be confusing that sometimes we see browser but sometimes not.

Reply:
But maybe it is good logic and general:
If given query is not from system scope and it is empty then we should also check system scope. And if both are empty we do not need a browser.